rio/src/RIO/Prelude/Logger.hs

```diff
@@ -292,11 +292,19 @@ logOptionsMemory = do
 -- to perform verbose logging or not. Individiual settings can be
 -- overridden using appropriate @set@ functions.
 --
+-- When Debug Flag is @True@:
+--
+--     * enables @logVerboseFormat@ option
```

Comment:
@roman Those are internal names, so might not be too informative to those who are new to the library. Because of this, do you think it could be better to reword them in such a way that the links to setters are also generated. Something along the lines:
```
-- When @Debug@ Flag is `True`, will modify all the options below:
--
--     * enables @`setLogVerboseFormat` `True`@
--     * enables @`setLogUseColor` `True`@, but only when the `Handle` is a terminal
--     * enables @`setLogUseLoc` `True`@
--     * enables @`setLogUseTime` `True`@
--     * sets @`setLogMinLevel` `Debug`@
```